rll/setup_automatic_upgrade.sh

```diff
@@ -40,12 +40,10 @@ EOF
   # Random minute 0-59
   cron_minute=$(( $RANDOM % 60 ))
 
-  umask 077
-
-  echo "${cron_minute} ${cron_hour} * * * root ${exec_file}" > ${cron_file}
+  sudo bash -c "umask 077 && echo '${cron_minute} ${cron_hour} * * * root ${exec_file}' > ${cron_file}"
```

Comment:
This should run as rightlink.  Since the rightlink agent is technically a package, let's keep it here and change 'root' to 'rightlink' instead of using the rightlink user's crontab:
`sudo bash -c "umask 077 && echo '${cron_minute} ${cron_hour} * * * rightlink ${exec_file}' > ${cron_file}"`

Since exec_file `rightlink_check_upgrade` is using rsc to schedule a recipe, no changes are needed there.